### DISCUSSION TITLE
Welcome to react-redux-firebase Discussions!

### DISCUSSION THREAD

**Discussion by prescottprue** [prescottprue/react-redux-firebase#1035] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by nemet4ndrea** [comment]
Hi! I'm Andrea, new to React & Firebase :). I want to say: react-redux-firebase is amazing. Thanks!

**Reply by prescottprue** [reply]
Thanks for reaching out to say so! Glad to hear it is helping! You might also want to check out [reactfire](https://github.com/FirebaseExtended/reactfire) if you aren't in need of having the data in redux
